File: age_distribution/age_distribution.rb
def age_categorizer(age)
  if (0..2).include? age
    puts("Still in Mama's arms")
  elsif (3..4).include? age
    puts("Preschool Maniac")
  elsif (5..11).include? age
    puts("Elementary school")
  elsif (12..14).include? age
    puts("Middle school")
  elsif (15..18).include? age
    puts("High school")
  elsif (19..22).include? age
    puts("College")
  elsif (23..65).include? age
    puts("Working for the man")
  elsif (66..100).include? age
    puts("The Golden Years")
  else
    puts("This program is for humans")
  end
end

f = File.open(ARGV[0], "r") do |file|
  file.each_line do |line|
    age_categorizer(line.to_i)
  end
end

File: age_distribution/age_distribution.py
import sys

def main(file_arg):
    with open(file_arg) as f:
        for line in f:
            age = int(line.rstrip())

            age_categorizer(age)


def age_categorizer(age):
    if age in range(0, 3):
        print("Still in Mama's arms")
    elif age in range(3, 5):
        print("Preschool Maniac")
    elif age in range(5, 12):
        print("Elementary school")
    elif age in range(12, 15):
        print("Middle school")
    elif age in range(15, 19):
        print("High school")
    elif age in range(19, 23):
        print("College")
    elif age in range(23, 66):
        print("Working for the man")
    elif age in range(66, 101):
        print("The Golden Years")
    else:
        print("This program is for humans")
            

if __name__ == '__main__':
    main(sys.argv[1])
